aodncore/pipeline/statequery.py

```diff
@@ -55,9 +42,9 @@ def query_wfs_urls_for_layer(self, layer, **kwargs):  # pragma: no cover
 
         :param layer: layer name supplied to GetFeature typename parameter
         :param kwargs: keyword arguments passed to underlying broker method
-        :return: list of files for the layer
+        :return: RemotePipelineFileCollection containing list of files for the layer
```

Comment:
A RemotePipelineFileCollection is really the default expectation for "query_" methods, so could be considered the "rule". Then in cases like `query_wfs_getfeature_dict`, the return type is an exception, so is named as such. Does that sound reasonable?

We could document better at the class level that RemotePipelineFileCollection is the standard response type, unless otherwise noted.

Reply:
:+1: That's fine.
I was just making the point that `_for_layer` is redundant because all WFS queries relate to a layer, and `_urls` is now plain wrong, because the method returns a RemotePipelineFileCollection rather than a list of URLs.
But I can see the benefit of _not_ renaming the method (less disruption in aodndata) so let's just leave it.

Reply:
Ah ok, I see what you mean now. Under the hood it is still dealing in URLs, extracting them from the layer, but at this API level it is "a collection of X for layer".

Its `features` the appropriate term here do you think, as a generic method name? Something as generic as `query_wfs_features` would be nice, given than it can be implicitly assumed that the return type is a collection.

Reply:
Update to use 'files' terminology